coinbase_test_suite.py

```diff
@@ -0,0 +1,14 @@
+import unittest
```

Comment:
I know you don't want a test suite with this branch. I will add this to .gitignore after this PR to ensure it doesn't merge with project. Since I chose a random product for the testProductsLink test above I wanted to show you how I would run the test 10 times in the test suite. Please don't get bothered about this being here for now :-D